system-tests/projects/e2e/cypress/e2e/proxying.cy.js

```diff
@@ -11,7 +11,8 @@ describe('proxying', () => {
       const script = document.createElement('script')
 
       script.src = 'https://localhost:7878/static/simple_obstructive_code.js'
-      script.integrity = 'sha256-iVKZPZrzbe7YNdMKYWJ1+f74j5lD3gRFvGjqtLyji6A='
+      // to regenerate: `openssl dgst -sha384 -binary **/static/simple_obstructive_code.js | openssl base64 -A`
+      script.integrity = 'sha256-BLwfivLJLqn8sjt9PEEXBlw6lO+DSZzFZ0rP/SWai2o='
```

Comment:
why did this change?

Reply:
`simple_obstructive_code.js` was linted and so the integrity hash changed

Reply:
also just realizing i made a slight typo here

```suggestion
      // to regenerate: `openssl dgst -sha256 -binary **/static/simple_obstructive_code.js | openssl base64 -A`
      script.integrity = 'sha256-BLwfivLJLqn8sjt9PEEXBlw6lO+DSZzFZ0rP/SWai2o='
```